### DISCUSSION TITLE
Copilot chat textbox goes off screen

### DISCUSSION THREAD

**Discussion by KG-droid** [community/community#107443] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Does anyone else experience an issue when typing in the Copilot Chat in Visual Studio 2022, where as your text approaches the end of the textbox instead of immediately starting a new line, it continues to type off the edge of the screen where I cant see it, until I type another 5 characters, then it moves it to a new line and I can see it again. I've just typed jibberish here but you get the idea:

<img width="287" alt="image" src="https://github.com/community/community/assets/97749670/c2f4562c-8b4e-4efc-8267-a62f05e3d794">

**Comment by ZephyrusBlaze** [comment]
you can try moving your cursor using the right or left arrow keys (`>` or `<`) keys to navigate within the textbox and see the text that's off-screen. Additionally, you can use the `ctrl + >` or `ctrl + <` shortcuts to move more quickly.

**Reply by KG-droid** [reply]
That doesn't work either unfortunately, the cursor just moves off the edge of the screen like the text does.

**Comment by ayim** [comment, marked as answer]
I was able to repro. Filed a bug here: 
We're going to track using the issue on VS Feedback:
https://developercommunity.visualstudio.com/t/GHCP-Chat-text-input-box:-text-and-curso/10582982

**Reply by KG-droid** [reply]
Thanks :)

**Comment by Pudly123** [comment]
It sure does. it is unusable as of today. After about 50 lines of code, it moves further and further off of left side of screen and then freezes.
